### DISCUSSION TITLE
V9: Are the DataGrid and the DropDown usable?

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#26107] [Q&A] [answered]
I would like to start a new project with V9 and I would like to know if the following components are (semi) usable, and how much of the API you expect to change to the **Preview Components**

- DataGrid?
- DropDown?

Creating data driven applications would be very difficult without the components in the **Preview Components** section.

Would you please provide some insight?

**Comment by ling1726** [comment]
Hi @blendsdk, DataGrid will be entering stable phase soon this month, and we don't expect any major changes.

@smhigley  might know more about dropdown

**Reply by blendsdk** [reply]
Thank you

**Comment by micahgodbolt** [comment, marked as answer]
Dropdown is now stable. DataGrid is close.
